### PR TITLE
Storybook: add new config.js and per-story option instructions

### PR DESCRIPTION
**Pull request by timhaines**
Installation:
image

Per-Story Options:
image

### CODE REVIEW

**Comment by timhaines** [comment]
@joscha volunteered to review this (thanks @joscha!). Will wait for his review before merging.

File: docs/clients/javascript/react-storybook.md [hunk 1]
```diff
@@ -60,6 +70,27 @@ These options can be appended to the percy-storybook command in the script tag i
 * **debug** - Provides additional debugging information. eg. `--debug`
 
 
+## Per-Story Options
+
+You can override options on a per-story basis by adding stories with `addWithPercyOptions` instead of `add`.
+
+`addWithPercyOptions` takes 3 parameters: the story name, the percyOptions, and the story fucntion.  If you've been calling `add` to add the story previously, simply change it to `addWithPercyOptions` and insert percyOptions as the second parameter.
```
Comment (joscha) [inline docs/clients/javascript/react-storybook.md]: Small typo here -> `fucntion`

File: docs/clients/javascript/react-storybook.md [hunk 2]
```diff
@@ -60,6 +70,27 @@ These options can be appended to the percy-storybook command in the script tag i
...
+
+Available options:
+
+* **widths** - An array of widths as integers for this story.
+* **rtl** - A boolean value specifying whether this story should additionally be run in a RTL direction.
```
Comment (joscha) [inline docs/clients/javascript/react-storybook.md]: `Pleae note: this option will trump the regex match for RTL snapshots, e.g. if you pass `false` here, Percy won't take an RTL snapshot even if the regular expression matches the story`

File: docs/clients/javascript/react-storybook.md [hunk 3]
```diff
@@ -60,6 +70,27 @@ These options can be appended to the percy-storybook command in the script tag i
...
 
+## Per-Story Options
+
+You can override options on a per-story basis by adding stories with `addWithPercyOptions` instead of `add`.
+
+`addWithPercyOptions` takes 3 parameters: the story name, the percyOptions, and the story fucntion.  If you've been calling `add` to add the story previously, simply change it to `addWithPercyOptions` and insert percyOptions as the second parameter.
+
+Available options:
+
+* **widths** - An array of widths as integers for this story.
```
Comment (joscha) [inline docs/clients/javascript/react-storybook.md]: `Given widths are cumulative. Percy will take a snapshot of the story with the given widths additionally to the widths specified for all stories`

File: docs/clients/javascript/react-storybook.md (hunk 2 above)
Comment (joscha) [inline docs/clients/javascript/react-storybook.md]: *Please

File: docs/clients/javascript/react-storybook.md [hunk 4]
```diff
@@ -60,6 +70,27 @@ These options can be appended to the percy-storybook command in the script tag i
...
+* **rtl** - A boolean value specifying whether this story should additionally be run in a RTL direction.
+
+Example:
+```javascript
+storiesOf('My basic span', module)
+  .addWithPercyOptions('with multiple widths', { widths: [222, 333], rtl: true }, () =>
+    <span>This span renders in at widths of 222px and 333px</span>,
```
Comment (joscha) [inline docs/clients/javascript/react-storybook.md]: Superfluous `at`

File: docs/clients/javascript/react-storybook.md (hunk 3 above)
Comment (joscha) [inline docs/clients/javascript/react-storybook.md]: I just noticed that this was my understanding of how the extra widths worked - is that even correct or do the given widths per story override the defaults?

File: docs/clients/javascript/react-storybook.md (hunk 1 above)
Comment (joscha) [inline docs/clients/javascript/react-storybook.md]:
```
Have a look at the storybook in [react-percy](https://github.com/percy/react-percy/tree/master/integration-tests/react-percy-storybook) for a real-world example and also how to use `.addWithPercyOptions` in conjunction with other add-ons.
```

File: docs/clients/javascript/react-storybook.md (hunk 3 above)
Comment (timhaines) [inline docs/clients/javascript/react-storybook.md]: @joscha The ones provided in the options override the defaults.  So in our integration tests, when you specified 222, snapshots are no longer taken at the regular widths.

Comment (joscha) [inline docs/clients/javascript/react-storybook.md]: Got it. I guess then we want:
```
Please note: Given widths override the default story widths.
```

Comment (joscha) [inline docs/clients/javascript/react-storybook.md]: Also, do you know where the default of `1280` is specified? We should probably reference this here.

**Pull request merged**